sway/commands/move.c

```diff
@@ -556,7 +556,7 @@ static struct cmd_results *cmd_move_container(int argc, char **argv) {
 			focus = seat_get_focus_inactive(seat, &old_ws->node);
 		}
 	}
-	seat_set_focus_warp(seat, focus, true, false);
+	seat_set_focus(seat, focus);
```

Comment:
I believe this is causing an extraneous workspace::focus event from the workspace that the container was moved to back to the original workspace.